Adding guide about Synthetics usage monitoring (#10766)
DataDog/documentation: content/en/synthetics/guide/monitor-usage.md

---
title: Monitor your Synthetics usage
kind: guide
further_reading:
    - link: '/synthetics/api_tests'
      tag: 'Documentation'
      text: 'Create an API Test'
    - link: '/synthetics/multistep'
      tag: 'Documentation'
      text: 'Create a Multistep API Test'
    - link: '/synthetics/browser_tests'
      tag: 'Documentation'
      text: 'Create a Browser Test'
    - link: 'https://www.datadoghq.com/blog/test-creation-best-practices/'
      tag: 'Blog'
      text: 'Best practices for creating end-to-end tests'

---

> Synthetic tests come with [estimated usage metrics][1] that allows you to keep track of your usage. These metrics notably enable you to:

Reviewer: ```suggestion
Synthetic tests come with [estimated usage metrics][1] that allow you to keep track of your usage. These metrics notably enable you to:
```

> * Understand how your usage evolves over time
> * Visualize which teams, applications, or services are contributing the most to your Synthetics usage and have the largest coverage

Reviewer: ```suggestion
* Understand how your usage evolves over time.
* Visualize which teams, applications, or services are contributing the most to your Synthetics usage and have the largest coverage.
```

* Alert on unexpected usage spikes that can impact your billing.

To visualize or alert on your Synthetics usage, use the following queries:

> * [Single][2] and [Multistep API tests][3]: `sum:datadog.estimated_usage.synthetics.api_test_runs{*}.as_count()`

Author: ```suggestion
* [Single][2] and [Multistep API tests][3]: `sum:datadog.estimated_usage.synthetics.api_test_runs{*}.as_count().`
```

> * [Single][2] and [Multistep API tests][3]: `sum:datadog.estimated_usage.synthetics.api_test_runs{*}.as_count().`

Author: ```suggestion
* [Single][2] and [Multistep API tests][3]: `sum:datadog.estimated_usage.synthetics.api_test_runs{*}.as_count()`
```

* [Browser tests][4]: `sum:datadog.estimated_usage.synthetics.browser_test_runs{*}.as_count()`.

> You can also scope or group these metrics by any tags associated with your test such as `team` or `application` for a higher level of refinement. 

Reviewer: ```suggestion
For a higher level of refinement, scope or group these metrics by tags associated with your test, such as `team` or `application`. 
```

---
title: Monitor your Synthetics usage
kind: guide
further_reading:
    - link: '/synthetics/api_tests'
      tag: 'Documentation'
      text: 'Create an API Test'
    - link: '/synthetics/multistep'
      tag: 'Documentation'
      text: 'Create a Multistep API Test'
    - link: '/synthetics/browser_tests'
      tag: 'Documentation'
      text: 'Create a Browser Test'
    - link: 'https://www.datadoghq.com/blog/test-creation-best-practices/'
      tag: 'Blog'
      text: 'Best practices for creating end-to-end tests'

---

> Synthetic tests come with [estimated usage metrics][1] allowing you to keep track of your usage. These metrics notably enable you to:

Author: ```suggestion
Synthetic tests come with [estimated usage metrics][1] that allows you to keep track of your usage. These metrics notably enable you to:
```

* Understand how your usage evolves over time
* Visualize which teams, applications, or services are contributing the most to your Synthetics usage and have the largest coverage
* Alert on unexpected usage spikes that can impact your billing.

To visualize or alert on your Synthetics usage, use the following queries:

* [Single][2] and [Multistep API tests][3]: `sum:datadog.estimated_usage.synthetics.api_test_runs{*}.as_count()`

* [Browser tests][4]: `sum:datadog.estimated_usage.synthetics.browser_test_runs{*}.as_count()`.

You can also scope or group these metrics by any tags associated with your test such as `team` or `application` for a higher level of refinement. 

> You can graph and monitor these metrics against static thresholds as well as machine learning based algorithms like [anomaly detection][5] or [forecast][6] to ensure you do not get alerted for expected usage growth.

Author: ```suggestion
You can graph and monitor these metrics against static thresholds as well as use machine learning based algorithms like [anomaly detection][5] or [forecast][6] to ensure you do not get alerted for expected usage growth.
```
